modules/setup-bias-monitoring.adoc

```diff
@@ -0,0 +1,83 @@
+:_module-type: PROCEDURE
+
+[id="setup-bias-monitoring_{context}"]
+= Before you begin
+
+== Downloading tutorial files
+. Go to https://github.com/trustyai-explainability/odh-trustyai-demos/tree/main.
+. Click the *Code* button and then click *Download ZIP* to download the repository.
+. Extract the downloaded repository files.
+
+== Enabling user-workload-monitoring
+To get enable Open Data Hub’s monitoring stack, you must enable monitoring for user-defined projects.
+
+. In a terminal window, if you are not already logged in to your OpenShift cluster as a cluster administrator, log in to the OpenShift CLI:
+... In the upper-right corner of the OpenShift web console, click your user name and select *Copy login command*.
+... After you have logged in, click *Display token*.
+... Copy the *Log in with this token* command and paste it in the OpenShift command-line interface (CLI).
++
+[source]
+----
+oc login --token=<token> --server=<openshift_cluster_url>
+----
+
+. To enable user workload monitoring, run the following command from the directory containing the downloaded tutorial files (`odh-trustyai-demos-main`):
++
+[source]
+----
+oc apply -f 1-Installation/resources/enable_uwm.yaml
+----
+
+. Configure user-workload-monitoring to hold metric data for 15 days: 
++
+[source]
+----
+oc apply -f 1-Installation/resources/uwm_configmap.yaml
+----
+
+Depending on how your cluster was created, you may need to enable a User Workload Monitoring setting from your {openshift-platform} (for example, on link:console.redhat.com[console.redhat.com]).
+
+== Enabling the TrustyAI component
+
+If you don’t already have an instance of {productname-long}, follow the steps in link:{odhdocshome}/installing-open-data-hub/#installing-odh-v2_installv2[Installing Open Data Hub]. After you have installed {productname-short}, you must enable the TrustyAI component in {productname-short}. 
+
+. Log in to {openshift-platform} as a cluster administrator.
+. In the {openshift-platform} console, click *Operators* -> *Installed Operators*.
+. Search for the *Open Data Hub Operator*, and then click the Operator name to open the Operator details page.
+. Click the *Data Science Cluster* tab.
+. Click the default instance name (for example, *default-dsc*) to open the instance details page.
+. Click the *YAML* tab to show the instance specifications.
+. In the `spec:components` section, set the `managementState` field for the `trustyai` component to `Managed`:
++
+----
+ trustyai:
+    managementState: Managed
+----
+. Click *Save*.
+
+
+== Setting up a project 
+. Create a new project named `model-namespace`. This name is arbitrary, but  will be used throughout the rest of this demo:
++
+[source]
+----
+oc new-project model-namespace
+----
+
+. Prepare the `model-namespace` for Open Data Hub's model serving: 
++
+[source]
+----
+oc label namespace model-namespace "modelmesh-enabled=true" --overwrite=true
+----
+
```

Comment:
Note
The monitoring models for bias (TrustyAI feature) is supported only on a multi-model serving platform that is based on the ModelMesh component. It is not supported on a single-model serving platform that is based on the KServe component.